ajax/libs/chirashi/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "chirashi",
+  "version": "5.1.5",
+  "description": "Minimalist DOM and events manager focused on performances",
+  "filename": "chirashi.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/chirashijs/chirashi.git"
+  },
+  "author": {
+    "name": "Alex Toudic @AlexToudic"
+  },
+  "keywords": [
+    "DOM",
+    "events"
+  ],
+  "npmName": "chirashi",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
You should exclude `.gitkeep` and `chirashi.min.js.gz`
Ref: https://github.com/isaacs/node-glob#glob-primer